src/modules/commander/Commander.cpp

```diff
@@ -426,18 +426,15 @@ transition_result_t Commander::arm(arm_disarm_reason_t calling_reason, bool run_
 
 	if (run_preflight_checks) {
 		if (_vehicle_control_mode.flag_control_manual_enabled) {
-			const bool throttle_above_low = (_manual_control_setpoint.z > 0.1f);
-			const bool throttle_above_center = (_manual_control_setpoint.z > 0.6f);
-
-			if (_vehicle_control_mode.flag_control_climb_rate_enabled && throttle_above_center) {
-				mavlink_log_critical(&_mavlink_log_pub, "Arming denied! Throttle not centered");
+			if (_vehicle_control_mode.flag_control_climb_rate_enabled && _manual_control.isThrottleAboveCenter()) {
+				mavlink_log_critical(&_mavlink_log_pub, "Arming denied because throttle above center");
```

Comment:
```suggestion
				mavlink_log_critical(&_mavlink_log_pub, "Arming denied: throttle above center");
```
Saving a few chars without loosing clarity.

Reply:
That's right, clear != long d909125